### DISCUSSION TITLE
Is RX7600XT supported on ROCm SDK for Windows?

### DISCUSSION THREAD

**Discussion by han-minhee** [ROCm/ROCm#3064] [Q&A] [answered]
AMD Radeon RX 7600 is marked as supported on the windows support list
What about the XT one?

**Comment by garrettbyrd** [comment]
I am not sure about Windows support, but as you can see from [this issue](https://github.com/ROCm/ROCm/issues/2972), there is not a lot of interest in fixing documentation inconsistencies regarding the 7600XT.

**Comment by darren-amd** [comment, marked as answer]
Hi @han-minhee,

We will begin supporting the 7600 XT in future releases, and will be adding it to the documentation soon, thanks for the question!

**Reply by han-minhee** [reply]
Good to see it coming! Thanks! Wish the ROCm support for next generation cards comes soon too😁
